### PR TITLE
Reader feature

### PR DESCRIPTION
**Pull request by titor-oopart**
I implemented ConfigFileReader that gets data from external file please let me know yours corrections thanks.

### CODE REVIEW

File: PivotalTracker.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+url=https://www.pivotaltracker.com/
+
+implicitWaitChrome=30
+explicitWaitChrome=40
+
```
Comment (Cxristian) [inline PivotalTracker.properties]: do you think that we need two differents waits for the differents browsers

Comment (titor-oopart) [reply PivotalTracker.properties]: Maybe we need run two scenarios in the same time with differents time waits.
Do you think that it's unncessary?

File: src/main/java/core/selenium/utils/ConfigFileReader.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        BufferedReader reader;
+        try {
+            reader = new BufferedReader(new FileReader(propertyFilePath));
+            properties = new Properties();
+            try {
+                properties.load(reader);
+                reader.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
```
Comment (LimbertVargas) [inline src/main/java/core/selenium/utils/ConfigFileReader.java]: maybe you put a Throw exception or this catch

**Review by LimbertVargas — COMMENTED** [review]
please fix your observations

File: PivotalTracker.properties (hunk 1 above)
Comment (titor-oopart) [reply PivotalTracker.properties]: I delete this waits you right

File: src/main/java/core/selenium/utils/ConfigFileReader.java (hunk 2 above)
Comment (titor-oopart) [reply src/main/java/core/selenium/utils/ConfigFileReader.java]: I change the exception

File: src/main/java/core/selenium/utils/CredentialsReader.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import java.io.FileReader;
+
+/**
+ * CredentialsReader class.
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class CredentialsReader {
+    final String JSONPATH = "credentials.json";
```
Comment (Cxristian) [inline src/main/java/core/selenium/utils/CredentialsReader.java]: this variable must be private...

Comment (titor-oopart) [reply src/main/java/core/selenium/utils/CredentialsReader.java]: done

**Review by aburgosu — APPROVED** [review]

**Review by LimbertVargas — APPROVED** [review]
it's ok John

**Pull request merged**